### PR TITLE
Update of config files to includes sidebar menu by default

### PR DESCRIPTION
**Pull request by alexander-fedorenko**
Missing part of the changes related to the updated layout. 
With the applied changes sidebar menu will be active by default for desktop on map viewer.
Context wizard plugin dependencies are updated.

### CODE REVIEW

File: configs/pluginsConfig.json [hunk 1]
```diff
@@ -555,32 +598,45 @@
...
+      ]
+    },
+    {
       "name": "StreetView",
       "glyph": "road",
       "title": "plugins.StreetView.title",
       "description": "plugins.StreetView.description",
-      "dependencies": ["Toolbar"],
+      "dependencies": [
+        "Toolbar"
```
Comment (offtherailz) [inline configs/pluginsConfig.json:629]: I think it needs to be modified this way.
```suggestion
        "SidebarMenu"
```

**Review by offtherailz — CHANGES_REQUESTED** [review]
Accordingly to the migration [guidelines](https://mapstore.readthedocs.io/en/latest/developer-guide/mapstore-migration-guide/#replacing-burgermenu-with-sidebarmenu)
- [ ] StreetView dependencies have to be changed into SidebarMenu
- [ ] BurgerMenu have to be visible

**Review by offtherailz — APPROVED** [review]

**Comment by landryb** [comment]
@alexander-fedorenko is it intended to still use the `BurgerMenu` plugin in geostory/dashboards/mobile views ?

**Comment by tdipisa** [comment]
> @alexander-fedorenko @tdipisa is it intended to still use the `BurgerMenu` plugin in geostory/dashboards/mobile views ?

yes

> **edit**: while here i noticed that the `About` plugin has `"showIn": ["BurgerMenu"]` in its config, unsure if that's also expected

where sorry?

**Comment by alexander-fedorenko** [comment]
@tdipisa it's in localConfig.json
@landryb Thanks, you're right, this setting makes "About" button appear only when BurgerMenu is in use. We'll update localConfig.json to fix this behavior.

**Pull request merged**